DFC.ServiceTaxonomy.Editor/Startup.cs

```diff
@@ -11,15 +10,14 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddOrchardCms();
         }
-        
+
         public void Configure(IApplicationBuilder app, IHostEnvironment env)
         {
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
             }
 
-            app.UseStaticFiles();
```

Comment:
We'll probably need to serve static files for the visualisation module, although the PR for that could add it back if it needs it.

Reply:
I removed as I vaguely remembered adding it for something I was playing around with, don't think it's needed right now!